### PR TITLE
Include a reference link to docs in generated fly.toml

### PR DESCRIPTION
**Pull request by dangra**
```toml
# fly.toml app configuration file generated for mounts-per-group on 2023-04-17T22:03:13Z
#
# See https://fly.io/docs/reference/configuration/ for reference documentation about this file
#
app = "mounts-per-group"
primary_region = "iad"

[http_service]
  internal_port = 8080
  force_https = true
  processes = ["app"]

[[mounts]]
  source = "webcache"
  destination = "/cache"
  processes = ["app"]

[[mounts]]
  source = "video_stage"
  destination = "/stage"
  processes = ["video"]

[processes]
  app = "/start webserver"
  notifier = "/start notifier"
  video = "/start video-converter"
```

### CODE REVIEW

File: internal/appconfig/serde.go [hunk 1]
```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml file generated for %s on %s\n\n", c.AppName, time.Now().Format(time.RFC3339))
```
Comment (alichay) [inline internal/appconfig/serde.go:18]: there's interpolation here that got encoded into the raw string on accident

**Review by alichay — CHANGES_REQUESTED** [review]

File: internal/appconfig/serde.go (hunk 1 above)
Comment (catflydotio) [inline internal/appconfig/serde.go:18]: How about 
```
fly.toml app configuration file generated for mounts-per-group on 2023-04-17T22:03:13Z
```
to make it more obvious what `fly.toml` is for?

File: internal/appconfig/serde.go [hunk 2]
```diff
@@ -15,6 +15,12 @@ import (
...
+#
+# See https://fly.io/docs/reference/configuration/ for reference documentation about this file
```
Comment (catflydotio) [inline internal/appconfig/serde.go:20]:
```suggestion
# See https://fly.io/docs/reference/configuration/ for  information about how to use this file.
```

Just looking for ways to connect more pathways in peoples' brains

File: internal/appconfig/serde.go (hunk 1 above)
Comment (dangra) [reply internal/appconfig/serde.go:18]: nice catch

Comment (dangra) [reply internal/appconfig/serde.go:18]: sounds good to me. you rocks.

File: internal/appconfig/serde.go (hunk 2 above)
Comment (dangra) [reply internal/appconfig/serde.go:20]: that is perfect. thanks

File: internal/appconfig/serde.go [hunk 3]
```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml app configuration file generated for %s on %s
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+`
+
```
Comment (alichay) [inline internal/appconfig/serde.go:23]: This is *such* a nitpick and it doesn't _really_ matter but IMO it looks nicer when there's a new line between the header comment and the content. This is a really nice change!
```suggestion
const flytomlHeader = `# fly.toml app configuration file generated for %s on %s
#
# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
#

`

```

**Review by alichay — APPROVED** [review]

**Pull request merged**